x-pack/plugins/observability_solution/observability/server/assets/component_templates/slo_summary_mappings_template.ts

```diff
@@ -79,6 +79,9 @@ export const getSLOSummaryMappingsTemplate = (
                 type: {
                   type: 'keyword',
                 },
+                params: {
```

Comment:
We only need this on the summary document since we don't have access to the saved objects for the SLO definition. We'd only put this in the rollup data if we needed to group-by.